project/api/models/representative.py

```diff
@@ -50,11 +48,27 @@ class Representative(models.Model):
     def __str__(self):
         return "Rep - {}".format(self.name)
 
-    def summarize(self):
-        return {
+    def summarize(self, account=None):
+        from .vote import Vote
+
+        data = {
             "bioguide_id": self.bioguide_id,
             "name": self.name,
             "party": self.party,
             "state": '{state}'.format(state=dict(US_STATES).get(self.state, self.state)),
             "profile_image": "https://theunitedstates.io/images/congress/225x275/{}.jpg".format(self.bioguide_id)
         }
+        if account:
+            votes = {}
+            bills = account.get_voted_bills(serialize=False)
+            supported_query = Vote.objects.filter(bill__in=bills["supported_bills"], representative=self)
+            opposed_query = Vote.objects.filter(bill__in=bills["opposed_bills"], representative=self)
+            votes["supported_the_same"] = supported_query.filter(vote="yes").count()
+            votes["supported_different"] = supported_query.filter(vote="no").count()
+
+            votes["opposed_the_same"] = opposed_query.filter(vote="no").count()
+            votes["opposed_different"] = opposed_query.filter(vote="yes").count()
```

Comment:
Can we add `voted_same` and `voted_different` counts here? I think that is what the user is ultimately after, to see how much they have in common with a representative.

- voted same is sum of
    - supported the same
    - opposed the same
- voted different is sum of
    - supported different
    - opposed different

It is also a bit more natural of language to say "`Representative` *voted the same* as `User` this many times".

Reply:
yeah sure, makes sense - didn't know how to handle it -> your way is great!